tools/logCollection/rawLogs/sample_tdr_logs.json

```diff
@@ -0,0 +1,30 @@
+[
```

Comment:
Same thing, do we want to include this or mark as ignored

Reply:
Ditto!

Reply:
I think the example input would be helpful for debugging if the users run into issues with the scripts, like if the tdr logs change format. But, I can add the rest of the folder to gitignore